docs/glossary.md

```diff
@@ -0,0 +1,100 @@
+# InSpec Glossary
+
+## Basic Syntax
+```
+describe foo('/path/to/foo.txt') do
+    its('blah') { should cmp '123' }
+    it { should exist }
+    it { should be_reasonable }
+    it { should_not be_ridiculous }
+end
+```
+## Basic Elements:
+
+### describe **foo**, where
+
+  * `foo` is the _resource_
+
+### describe foo **('/path/to/foo.txt')**, where
+
+  * `('/path/to/foo.txt')` is the _resource parameter_
+
+## Tests:
+
+### **its('blah') { should cmp '123' }** is an _individual test_, where
+
+  * `'blah'` is a _property_
+  * { should cmp '123' }is a _condition statement_
+  * `should`  is the _condition_
+  * `cmp`  is the _matcher_
+  * `123`  is the _expected result_
```

Comment:
Technically, since the example has `123` wrapped in quotes, the expected result is actually `'123'` and not `123` - string vs. integer. This should probably be:

```
 * `'123'` is the _expected result_
```

Reply:
OK